plotly/graph_objs/graph_objs.py

```diff
@@ -0,0 +1 @@
+from plotly.graph_objs import *
```

Comment:
this change seems odd to me, seems like something that should exist (if necessary) in https://github.com/plotly/plotly.py/pull/942. What's the rational behind this change?

Reply:
I actually did add this in #942 as well. It is a bit weird, but some tests (and examples I've seen online) have something like `from plotly.graph_objs import graph_objs` (e.g. https://plot.ly/python/scatterplot-matrix/)

It's an easy enough thing to keep backward compatible, but do we want to keep it around or deprecate it?

Reply:
Yeah, that's odd. It seems like the more semantic way to write this would be 
```
import plotly.graph_objs as graph_objs
```
or, the way we've been doing it all along
```
import plotly.graph_objs as go
```
In those examples (https://plot.ly/python/scatterplot-matrix/) `graph_objs` isn't even used.

OK, well let's keep it for backwards compatibility but I'd be OK with deprecating it.